deployment/legacy-migration/maintenance-worker/migration/neo4j/users/users.cql

```diff
@@ -117,7 +117,7 @@ MERGE (e:EmailAddress {
   verifiedAt: toString(datetime())
 })
 MERGE (e)-[:BELONGS_TO]->(u)
-MERGE (u)<-[:PRIMARY_EMAIL]-(e)
+MERGE (u)-[:PRIMARY_EMAIL]->(e)
```

Comment:
is this code creep?
I realised that this was incorrect during the last deploy to production, but never put in a `PR` to fix it... 
I can separate it into another `PR`, if y'all prefer

Reply:
Yes, it is unrelated! However it's surprising to me that this was not found earlier.